core/node/eth_sender/src/eth_tx_aggregator.rs

```diff
@@ -1147,6 +1156,35 @@ impl EthTxAggregator {
 
         GatewayMigrationState::from_sl_and_notification(self.settlement_layer, notification)
     }
+
+    async fn is_waiting_for_batches_with_interop_roots_to_be_committed(
+        &self,
+        storage: &mut Connection<'_, Core>,
+    ) -> Result<bool, EthSenderError> {
+        let latest_processed_l1_batch_number = storage
+            .interop_root_dal()
+            .get_latest_processed_interop_root_l1_batch_number()
+            .await?;
+
+        if latest_processed_l1_batch_number.is_none() {
+            return Ok(true);
+        }
+
+        let last_sent_successfully_eth_tx = storage
+            .eth_sender_dal()
+            .get_last_sent_successfully_eth_tx_by_batch_and_op(
+                L1BatchNumber::from(latest_processed_l1_batch_number.unwrap()),
+                L1BatchAggregatedActionType::Commit,
+            )
+            .await
+            .unwrap();
```

Comment:
why `unwrap`? 
This means, that eth tx has been created but not yet send, so it's actually `true`